pallets/usdv-minting/src/lib.rs

```diff
@@ -13,14 +12,20 @@ mod tests;
 
 type BalanceOf<T> =
     <<T as Trait>::Currency as MultiCurrency<<T as frame_system::Trait>::AccountId>>::Balance;
+type MintMembers = pallet_membership::Instance0;
```

Comment:
So if moving minting to the provider pallet we would still have this membership as a configurable associated type that is initially also set to the default instance so every provider can be minter initially but wouldn't always be the case as all minters are providers but not all providers are minters.